### PR TITLE
Test for cassandra duplicate table creation bug

### PR DESCRIPTION
**Pull request by hsaraogi**
This is a test with multiple Cassandra nodes to find if we trigger multiple table creation. This is Ignored as it needs some manual verification.

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/753)

### CODE REVIEW

**Comment by joelea** [comment]
Lets get this running in an automated fashion and then we'll merge it.

**Comment by jboreiko** [comment]
Would like to get this down to one owner and know what the next steps are going to be to get this automated.

File: atlasdb-cassandra-multinode-tests/src/test/java/CassandraSchemaLockTest.java [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService;
+import com.palantir.docker.compose.DockerComposition;
+import com.palantir.docker.compose.connection.DockerPort;
+import com.palantir.docker.compose.connection.waiting.HealthCheck;
+import com.palantir.docker.compose.connection.waiting.SuccessOrFailure;
+
+public class CassandraSchemaLockTest {
+
+    public static final int THRIFT_PORT_NUMBER = 9160;
+    @ClassRule
```
Comment (joelea) [inline atlasdb-cassandra-multinode-tests/src/test/java/CassandraSchemaLockTest.java]: This has the same problem that was fixed in #782

**Comment by jboreiko** [comment]
Doesn't appear that this was ever run with the correct amount of parallelism (ie the passing test didn't actually run this code from what I can tell).  I've bumped us up to 6 containers and kicked off a build of develop - https://circleci.com/gh/palantir/atlasdb/1976

**Pull request merged**